llm_bioinformatics_research/src/App.js

```diff
@@ -23,11 +24,11 @@ const Layout = ({ children }) => {
           </Typography>
 
           <Box sx={{ flexGrow: 1, display: 'flex', justifyContent: 'flex-end' }}>
-            <Button component ={Link} to="/home" sx={{ color: 'white' }}>Home</Button>
+            <Button component ={Link} to="/home" sx={{ bgcolor: location.pathname === '/home' ? grey[500] : blue[500], color: 'white', m: 1 }}>Home</Button>
```

Comment:
I think the grey highlight on the buttons for signaling the current page doesn't fit the theme of the website quite well. Do you think it would be better to slightly change the tone of blue that is being used instead of using the grey? I think there is a css feature that can accommodate this.